.gitignore

```diff
@@ -5,3 +5,4 @@ secrets.js
 .DS_Store
 npm-debug.log
 yarn-error.log
+script/seed.js
```

Comment:
why are you ignoring `script/seed`?

Reply:
Because we're adding admin access to the user model and set one of the users in the seed file to an admin, and the seed file has their username and password so we wouldn't want anyone to access that information on github.

Reply:
The seed file is for development. It doesn't contain actual secrets because you don't run it on production.

Making this gitignored means now you have to share the seed script manually and it isn't version controlled anymore.

If you need a seed script which does seed sensitive info, you should pull that info from the environment, which is customized in the secrets file.